lib/commands/commands/riot.ts

```diff
@@ -32,39 +34,157 @@ export const getSummonerId = async (ign:string, server:string) => {
-        log.INFO(path);
-        lastGame = await axios(path);
-    }
-    catch(err) {
+        : undefined;
+
+    const pathLastGameData = `https://${realm}.api.riotgames.com/lol/match/v4/matches/${matchId}?api_key=${config.RIOT_API_TOKEN}`;
+    const pathLastGameTimeline = `https://${realm}.api.riotgames.com/lol/match/v4/timelines/by-match/${matchId}?api_key=${config.RIOT_API_TOKEN}`;
+    const lastGameData:any = await axios(pathLastGameData).catch(err => {
+        log.WARN(err);
+        msg.channel.send(createEmbed(`❌Cannot get match data`, [{ title: '\_\_\_', content: `Fetching game ${matchId} data failed.` }]));
+        return;
+    })    
+    let lastGameTimeline:any = await axios(pathLastGameTimeline).catch(err =>{
         log.WARN(err);
-        msg.channel.send(createEmbed(`❌Cannot get match data`, [{ title: '\_\_\_', content: `Fetching match ${matchId} data failed.` }]));
+        msg.channel.send(createEmbed(`❌Cannot get match data`, [{ title: '\_\_\_', content: `Fetching timeline of game ${matchId} data failed.` }]));
         return;
+    })
+
+    let players:any = [ ];
+    let winningTeam:any = '';
+
+    lastGameData.data.teams.map((team:any) => {
+        if(team.win === 'Win')
+            winningTeam = team.teamId;
+    })
+    lastGameData.data.participantIdentities.map((participant:any) => {
+        players.push({
+            gameId: participant.participantId,
+            id: participant.player.accountId,
+            name: participant.player.summonerName,
+        })
+    })
+    lastGameData.data.participants.map((participant:any) => {
+        const i = players.findIndex((player:any) => player.gameId === participant.participantId);
+        players[i].win = participant.teamId === winningTeam;
+        players[i].teamId = participant.teamId;
+        players[i].championId = participant.championId;
+        players[i].champion = champions.find(champ => champ.id == participant.championId);
+        players[i].summ1 = participant.spell1Id;
+        players[i].summ2 = participant.spell2Id;
+        players[i].fbkill = participant.stats.firstBloodKill;
+        players[i].fbassist = participant.stats.firstBloodAssist;
+        players[i].role = participant.timeline.role;
+        players[i].lane = participant.timeline.lane;
+    })
+    const ourPlayer = players.find((player:any) => player.id === playerId);
+    const enemies = players.filter((player:any) => player.lane === ourPlayer.lane && player.teamId !== ourPlayer.teamId);
+    const lane = cache["lanes"].find((lane:any) => lane.lane == ourPlayer.lane)
+    const queue = cache["queues"].find((queue:any) => queue.id == lastGameData.data.queueId)
+
+    const embed = new Discord.RichEmbed()
+        .setColor('FDC000')
+        .setThumbnail(lane ? lane.icon : 'https://d1nhio0ox7pgb.cloudfront.net/_img/g_collection_png/standard/256x256/question.png')
```

Comment:
Could use a Discord one instead, less likely to randomly disappear.

Reply:
Does Discord have some kind of placeholder picture free to use?

Reply:
Oh it's the big image in the top-right corner? I misunderstood in that case.